actor-client/src/test/scala/sbt/client/actors/FakeSbtClient.scala

```diff
@@ -170,6 +170,10 @@ final class FakeSbtClient(refFactory: ActorRefFactory,
 
   def lookupScopedKey(name: String): Future[Seq[protocol.ScopedKey]] = scopedKeyLookup(name)
 
+  def listSettings(): Future[Seq[protocol.ScopedKey]] = ???
```

Comment:
I'm testing it in the integration tests. This fake client thing is used in the tests for the actor client. I haven't been adding new requests to the actor client.